### PR TITLE
Change Backlight

### PR DESCRIPTION
**Pull request by Detailzc**
This PR is used to make the backlight of WALLHREO adjustable

### CODE REVIEW

File: drivers/SmartThings/zigbee-switch/src/wallhero/init.lua [hunk 1]
```diff
@@ -92,6 +95,22 @@ local function device_added(driver, device)
...
+  local preferences = device.preferences
+  local old_preferences = args.old_st_store.preferences
+  local value_map = { [true] = 0x01,[false] = 0x00 }
+  if preferences ~= nil then
+    local id = "stse.turnOffIndicatorLight"
+    local old_value = old_preferences[id]
+    local value = preferences[id]
+    if value ~= nil and value ~= old_value  then
+      value = value_map[value]
+      local message = cluster_base.write_manufacturer_specific_attribute(device, 0x0006, 0x6000, 0x1235, data_types.Uint8, value)
```
Comment (liuyd96) [inline drivers/SmartThings/zigbee-switch/src/wallhero/init.lua:108]: Please use definitions from the development library.

```
local OnOff = zcl_clusters.OnOff
local MFG_CODE = 0x1235
local PRIVATE_ATTRIBUTE = 0x6000
local message = cluster_base.write_manufacturer_specific_attribute(device, OnOff.ID, PRIVATE_ATTRIBUTE , MFG_CODE , data_types.Uint8, value)
```

**Review by greens — APPROVED** [review]
This looks fine, but I'll note that users will need to re-add for these changes to appear for them.

**Comment by Detailzc** [comment]
This attachment is the results of our eight switch device self testing

**Comment by vangoran** [comment]
Self-verification is fine, so good to merge it

**Pull request merged**